imap_processing/ialirt/l0/process_swapi.py

```diff
@@ -137,14 +129,18 @@ def optimize_pseudo_parameters(
     return solution_dict
 
 
-def process_swapi_ialirt(unpacked_data: xr.Dataset) -> list[dict]:
+def process_swapi_ialirt(
+    unpacked_data: xr.Dataset, calibration_lut_table: pd.DataFrame
+) -> list[dict]:
     """
     Extract I-ALiRT variables and calculate coincidence count rate.
 
     Parameters
     ----------
     unpacked_data : xr.Dataset
         SWAPI I-ALiRT data that has been parsed from the spacecraft packet.
+    calibration_lut_table : pd.DataFrame
+        DataFrame containing the contents of the SWAPI lookup table.
```

Comment:
I would mention something about which calibration file is used here since we have couple LUT. 
```suggestion
        DataFrame containing the contents of the SWAPI esa-unit-conversion lookup table.
```